app/static/js/services/linc.auth.services.js

```diff
@@ -41,6 +41,7 @@ angular.module('linc.auth.services', [])
 			}
 			$localStorage.user = auth_user;
 			authService.user = auth_user;
+      $rootScope.$broadcast('refreshOrganizations');  // avoid problems with stale orgs
```

Comment:
might as well call this after login, in the event that person A logs out but has local data lying around, person B makes a change, and person A logs back in